### PR TITLE
Add tests for images.util

### PR DESCRIPTION
**Pull request by manuelblum**
## What

Adds Vitest tests for `images.util` in `@comet/cms-api` (`packages/api/cms-api/src/file-utils/images.util.ts`).

The module was previously untested despite being exported from the package's public API (`src/index.ts`). It contains non-trivial logic across three functions:

- **`getMaxDimensionsFromArea`** — aspect-ratio-aware dimension calculation with `Math.ceil` rounding, two distinct code paths (portrait vs landscape)
- **`getCenteredPosition`** — focal-point-based crop positioning with boundary clamping and area-offset arithmetic, covering all five `FocalPoint` enum values
- **`getSupportedMimeType`** — MIME-type negotiation via `@hapi/accept`

**Tests added** (`src/file-utils/__tests__/images.util.test.ts`):

| Suite | Cases |
|---|---|
| `getMaxDimensionsFromArea` | square 1:1, landscape ratio, ratio that overflows width, portrait constrained by height, portrait constrained by width, fractional ceiling |
| `getCenteredPosition` | CENTER, area offset, NORTHWEST boundary clamp, SOUTHEAST overflow clamp, NORTHEAST, SOUTHWEST, crop equals area |
| `getSupportedMimeType` | exact match, no match, empty accept, no match from multi-type header, negotiated best match |

All 299 tests in the package pass.

---
_Generated by [Claude Code](https://claude.ai/code/session_0183gxXbPeShppjGu5rzRoZe)_

### CODE REVIEW

File: packages/api/cms-api/src/file-utils/__tests__/images.util.test.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    });
+});
+
+describe("getSupportedMimeType", () => {
+    it("returns matching mime type when accept header includes it", () => {
+        expect(getSupportedMimeType(["image/webp", "image/jpeg"], "image/webp")).toBe("image/webp");
+    });
+
+    it("returns empty string when accept header does not include a supported type", () => {
+        expect(getSupportedMimeType(["image/webp"], "image/jpeg")).toBe("");
```
Comment (manuelblum) [inline packages/api/cms-api/src/file-utils/__tests__/images.util.test.ts]: thats kind of unintended behaviour in my opinion, it should return null.

Comment (manuelblum) [reply packages/api/cms-api/src/file-utils/__tests__/images.util.test.ts]: @johnnyomair any idea why we are returning here an empty string?

Comment (johnnyomair) [reply packages/api/cms-api/src/file-utils/__tests__/images.util.test.ts]: No. My best guess is because `mime.getExtension()` expects a string here: https://github.com/vivid-planet/comet/blob/ca47f0a2001c1e406c325f56301b61471c2533c2/packages/api/cms-api/src/dam/images/images.controller.ts#L226-L228
But you're right, it should return null or undefined.

**Review by johnnyomair — DISMISSED** [review]

File: packages/api/cms-api/src/file-utils/__tests__/images.util.test.ts (hunk 1 above)
Comment (manuelblum) [reply packages/api/cms-api/src/file-utils/__tests__/images.util.test.ts]: Adapted the function in 28539ec to return `undefined` instead of an empty string. Minimal changes — only the two internal callers needed a `?? ""` fallback so `mime.getExtension` still receives a `string`. The public API (`src/index.ts`) was not affected; `getSupportedMimeType` is internal.

**Comment by johnnyomair** [comment]
@manuelblum the test workflow fails.

**Comment by manuelblum** [comment]
> @manuelblum the test workflow fails.

needed to fix an edge case.

**Review by johnnyomair — APPROVED** [review]
